pkg/api/addons.go

```diff
@@ -167,7 +175,7 @@ func (cs *ContainerService) setAddonsConfig(isUpdate bool) {
 
 	defaultMetricsServerAddonsConfig := KubernetesAddon{
 		Name:    MetricsServerAddonName,
-		Enabled: k8sVersionMetricsServerAddonEnabled(o),
+		Enabled: to.BoolPtr(common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.9.0")),
```

Comment:
Carried this additional default gate over from the "last mile" "concatenate addons string" business logic area.